saint-app/db-postgres/src/saint_gateway/query.rs

```diff
@@ -12,3 +12,31 @@ pub async fn find_one_by_id(client: &Client, id: Uuid) -> Result<Row, Error> {
     let name_param: &[&(dyn ToSql + Sync)] = &[&id];
     client.query_one(&stmt, name_param).await
 }
+
+pub async fn get_collection(
+    client: &Client,
+    offset: Option<u16>,
+    count: Option<u16>,
+    is_male: Option<bool>,
+    display_name: Option<String>,
+) -> Result<Vec<Row>, Error> {
+    let statement = format!(
+        "SELECT * FROM saint__saint_view \
+        WHERE display_name LIKE {} AND is_male IN ({}) \
```

Comment:
See https://www.postgresql.org/docs/9.1/functions-comparison.html
```
Boolean values can also be tested using the constructs

expression IS TRUE
```

You should find a way to **exclude** `is_male` when the Option is None